geps/gep-1651.md

```diff
@@ -0,0 +1,154 @@
+the attributes `GatewayAddress.Scope` and `GatewayAddress.Type`.
+
+### Predefined Scopes
+
+Implementations MAY implement the following supplied scopes and MUST abide by 
+their defined accessibility
+
+Scope | Accessibility
+-|-
+`Public`|The address is accessible over the public internet
+`ClusterLocal`|The address is accessible only within the local cluster
+
+### Vendor prefixed scopes
+
+Implementations can define custom scopes by specifying a vendor prefix followed
+by a slash `/` and a custom scope name ie. `com.ingress.dev/my-scope`.
+
+### Default scope
+The default value of `scope` is implementation specific. It is RECOMMENDED that
+the default `scope` remains consistent for Gateways with the same
+`gatewayClassName`.
+
+
+## Examples
+
+#### 1. Request a GatewayAddress that is only accessible within a cluster
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: prod-web
+spec:
+  gatewayClassName: acme-lb
+  listeners:  
+  - protocol: HTTP
+    port: 80
+  addresses:
+  - scope: ClusterLocal
+```
+#### 2. Request a GatewayAddress with a specific scope and address
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: prod-web
+spec:
+  gatewayClassName: acme-lb
+  listeners:  
+  - protocol: HTTP
+    port: 80
+  addresses:
+  - scope: ClusterLocal
+    value: 10.0.0.8
+```
+#### 3. Request a GatewayAddress that is publically accessible
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: prod-web
+spec:
+  gatewayClassName: acme-lb
+  listeners:  
+  - protocol: HTTP
+    port: 80
+  addresses:
+  - scope: Public
+```
+
+## Alternatives
+
+### Introducing new GatewayAddress Types
+
+We could introduce additional `AddressTypes` (ie. `ClusterLocalIPAddress`) but
+this would lead to a combinatorial explosion as new dimensions (ie. IPv6) are
+introduced.
+
+See examples in this comment: https://github.com/kubernetes-sigs/gateway-api/pull/1653#issuecomment-1451246877
+
+### Survey of Prior Art
+
+These alternatives are a survey of existing approaches to support cluster
+local Gateways. Most are implementation specific and are not portable.
+
+#### Special annotation/label
+
+Istio let's you specify an annotation `networking.istio.io/service-type` to 
+change the underlying Kubernetes Service type to make it a ClusterIP type.
+
+#### Re-use of AddressType Hostname
+
+Istio let's you re-use existing Gateway deployments by setting the address
+type to `Hostname` and the value to the Istio ingress Kubernetes Service. If an 
+operator configures the Istio deployment to support cluster local traffic a 
+Gateway implementation can select it using the `HostName` attribute.
+
+#### Multiple Gateway Classes
+
+Some implementations support multiple deployments on a single cluster where
+each maps to a GatewayClass. One of these deployments can be configured to serve
+cluster local traffic.
```

Comment:
As I detailed in https://github.com/kubernetes-sigs/gateway-api/pull/1653/files#r1143956454, a single GatewayClass can support this use case through parametersRef.

Reply:
closing as a duplicate